src/pages/exclusive-content/components/active-view.js

```diff
@@ -13,15 +12,15 @@ export function ActiveView() {
   if (view === ViewStates.Generate) {
     return (
       <>
-        <SectionHeader>Generate Exclusive Content</SectionHeader>
```

Comment:
This component doesn't seem to be very necessary if we have chosen to standardise the baseline typographic CSS for headings.